src/SimulationStrategyFactory.py

```diff
@@ -0,0 +1,82 @@
+from enum import IntEnum
+from src.LossyBosonSamplingExactDistributionCalculators import BosonSamplingExperimentConfiguration
+from src.simulation_strategies.SimulationStrategy import SimulationStrategy
+from src.simulation_strategies.FixedLossSimulationStrategy import FixedLossSimulationStrategy
+from src.simulation_strategies.UniformLossSimulationStrategy import UniformLossSimulationStrategy
+from src.simulation_strategies.GeneralizedCliffordsSimulationStrategy import GeneralizedCliffordsSimulationStrategy
+from src.simulation_strategies.CliffordsRSimulationStrategy import CliffordsRSimulationStrategy
+
+
+class StrategyTypes(IntEnum):
+    FIXED_LOSS = 1
```

Comment:
Nitpick: Unless these integer values are important, one could use `enum.auto()` like in
```
class Enumerator(enum.IntEnum):
    VALUE_1 = enum.auto()
    VALUE_2 = enum.auto()
```